discord/message.py

```diff
@@ -1391,10 +1401,19 @@ async def edit(
             are used instead.
 
             .. versionadded:: 1.4
-        view: Optional[:class:`~discord.ui.View`]
+        view: Optional[Union[:class:`~discord.ui.View`, :class:`~discord.ui.LayoutView`]]
             The updated view to update this message with. If ``None`` is passed then
             the view is removed.
 
+            .. note::
+
+                If you want to update the message to have a :class:`~discord.ui.LayoutView`, you must
+                explicitly set to ``None`` or empty array, as required, the ``content``, ``embed``,
+                ``embeds``, and ``attachments`` parameters.
```

Comment:
```suggestion
                If you want to update the message to have a :class:`~discord.ui.LayoutView`, you must
                explicitly set the ``content``, ``embed``, ``embeds``, and ``attachments`` parameters to
                ``None`` if the previous message had any.
```

Do the same wording change for the rest of em too. The original sounds like broken English.